include/motor-control/core/tmc2130.hpp

```diff
@@ -126,6 +132,21 @@ class TMC2130 {
         return true;
     }
 
+    [[nodiscard]] auto convert_to_tmc2130_current_value(uint32_t c) const
+        -> uint32_t {
+        constexpr const float SMALL_R = 0.02;
+        const auto SQRT_TWO = static_cast<float>(std::sqrt(2));
+        auto FLOAT_CONSTANT = static_cast<float>(
+            SQRT_TWO * 32.0 * (_current_config.r_sense + SMALL_R) /
+            _current_config.v_sf);
+        auto fixed_point_constant = static_cast<uint32_t>(
+            FLOAT_CONSTANT * static_cast<float>(1LL << 16));
```

Comment:
we have functions that allow you to multiply and divide fixed point

Reply:
Will add this unsigned fixed point multiply func to that file